src/egon/data/datasets/emobility/motorized_individual_travel/__init__.py

```diff
@@ -414,12 +453,16 @@ def generate_model_data_tasks(scenario_name):
 
         super().__init__(
             name="MotorizedIndividualTravel",
-            version="0.0.4",
+            version="0.0.6",
             dependencies=dependencies,
             tasks=(
                 create_tables,
                 {
-                    (download_and_preprocess, allocate_evs_numbers),
+                    (
+                        download_and_preprocess,
+                        allocate_evs_numbers,
+                        write_metadata_to_db,
+                    ),
                     (extract_trip_file, write_evs_trips_to_db),
```

Comment:
The writing needs to be put after the trip file extraction as there won't be any meta file to read before.
(I guess it ran successfully on your instance because the trip file was already there?)
I'l fix this.

Reply:
done. ready for merge.